CMakeLists.txt

```diff
@@ -385,6 +385,28 @@ toolchain_ld_force_undefined_symbols(
 if(NOT CONFIG_NATIVE_BUILD)
   # @Intent: Set linker specific flags for bare metal target
   toolchain_ld_baremetal()
+
+  zephyr_link_libraries(PROPERTY baremetal)
+
+  # Note that some architectures will # skip this flag if set to error, even
```

Comment:
floating #

Reply:
pressing `J` in vim really ought to autoremove comment tags, such as `//` and `#` :laughing: 

Thanks for catching, and fixed.